dataland-frontend/src/components/forms/CreateEuTaxonomyForFinancials.vue

```diff
@@ -573,7 +579,7 @@ export default defineComponent({
       } catch (error) {
         this.messageCount++;
         console.error(error);
-        this.message = formatAxiosErrorMessage(error);
+        this.message = formatAxiosErrorMessage(error as Error);
```

Comment:
Was the linter the reason for this addition of a type?

Reply:
Yes

Reply:
ok